styles/index.css

```diff
@@ -0,0 +1,116 @@
+:root {
+    --main-bg-color: rgb(16, 3, 46);
+    --starwars-yellow: rgb(88%, 69%, 0%);
+    --logo-width: calc(150px * 1);
+    --nav-bar_height: 8em;
+}
+
+/* BACKGROUND */
+body {
+    background-color: var(--main-bg-color);
+    margin: 0;
+}
+
+.main-bg-image {
+    width: 100%;
+    height: 170%;
+    position: absolute;
+    background-position: center;
+    background-image: url("../images/background_stars.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    /* filter: grayscale(55%); */
+    backdrop-filter: var(--main-bg-color);
+    left: 0;
+}
+
+#main-containter {
+    display: grid;
+    align-items: center;
+    justify-items: center;
+    grid-template-columns: var(--logo-width) 1fr 1fr;
+    grid-template-rows: auto 1fr;
+    grid-template-areas:
+        "logo     bar     bar"
+        "content content content";
+}
+
+.sw-logo {
+    grid-area: logo;
+    align-self: start;
+    background-size: 100px;
+    width: 100px;
+    height: 50px;
+    background-image: url("../images/starwars_logo.png");
+    background-repeat: no-repeat;
+    margin-top: 1em;
+}
+
+/* QUIZ MENU */
+.quiz-main-menu {
```

Comment:
You can add property align-items: center to this class. Menu will look symmetrical at the mobile view.